AlfrescoApp/Model/Managers/RealmManager.m

```diff
@@ -61,7 +61,7 @@ - (void)createMainThreadRealmWithCompletionBlock:(void (^)(void))completionBlock
     else
     {
         dispatch_sync(dispatch_get_main_queue(), ^{
-            _mainThreadRealm = [self createRealmWithName:[AccountManager sharedManager].selectedAccount.accountIdentifier];
+            _mainThreadRealm = [[RealmSyncCore sharedSyncCore] realmWithIdentifier:[AccountManager sharedManager].selectedAccount.accountIdentifier];
```

Comment:
Using a instance variable within a block creates a strong reference to self. Use a property instead with "weakify" and "strongify" constructs